prisma/seeds/seed-notifications.ts

```
@@ -9,7 +9,13 @@
  */
 import { PrismaClient, NotificationCategory } from '@prisma/client';
 
-const prisma = new PrismaClient();
+if (!process.env.DIRECT_DATABASE_URL) {
+  throw new Error('DIRECT_DATABASE_URL environment variable is required for seeding');
```

ESLint

[error] Replace `'DIRECT_DATABASE_URL·environment·variable·is·required·for·seeding'` with `⏎····'DIRECT_DATABASE_URL·environment·variable·is·required·for·seeding',⏎··`

(prettier/prettier)